src/dataEntities/block.ts

```diff
@@ -0,0 +1,17 @@
+import { ethers } from "ethers";
+
+export interface IBlockStub {
+    hash: string;
+    number: number;
+    parentHash: string;
+}
+
+export interface HasLogs {
```

Comment:
I would just call LogList, as that's what it is. Same with HasTxHashes.

There's a TODO here that should be cleaned up. I would say stick with the ethers class for now.

Reply:
Changed to "Logs" and "Transactions".